splink/comparison_template_library.py

```diff
@@ -1385,6 +1385,7 @@ def __init__(
         term_frequency_adjustments_full: bool = False,
         include_exact_match_level: bool = True,
         include_username_match_level: bool = True,
+        include_username_fuzzy_level: bool = True,
```

Comment:
Minor nag: in future would it be possible to keep this sort of thing to a separate PR?

Reply:
Yep, that was v lazy of me. Converting to new functions led to a load of not trained level warnings so I put a fix in directly, but I am aware that it should not be something done in a PR like this.